features/ui/src/main/java/org/opennms/alec/rest/SituationRestImpl.java

```diff
@@ -56,9 +59,10 @@
 import com.fasterxml.jackson.databind.ObjectMapper;
 
 public class SituationRestImpl implements SituationRest {
-    private static final Logger LOG = LoggerFactory.getLogger(SituationRestImpl.class);
+    private static final Logger LOGGER = LoggerFactory.getLogger(SituationRestImpl.class);
```

Comment:
nit: why LOGGER instead of LOG? just curious. Not sure what the convention is in OpenNMS

Reply:
I don't remember why I changed this O_o, reverted